website/content/docs/agent/index.mdx

```diff
@@ -165,9 +165,10 @@ These are the currently-available general configuration option:
 There can at most be one top level `vault` block and it has the following
 configuration entries:
 
-- `address` `(string: <optional>)` - The address of the Vault server. This should
-  be a complete URL such as `https://127.0.0.1:8200`. This value can be
-  overridden by setting the `VAULT_ADDR` environment variable.
+- `address` `(string: <optional>)` - The address of the Vault server to connect.
+  This should be a complete URL such as `https://vault.example.org:8200` or
```

Comment:
Please do not use `vault.example.org`:
```
dig +short example.org
  # 93.184.216.34
```
As it's a top-level-domain (TLD) that could and does belong to someone - therefore any `*.example.org` could resolve.

Please use single word or hyphened word such as: `vault-fqdn` to ensure a none resolvable address unless unless local or otherwise something set via the DNS of the user.